develop/TOOLS/DIS/Installing_and_configuring/Prerequisites.md

```diff
@@ -23,6 +23,7 @@ DataMiner Integration Studio is a Microsoft Visual Studio extension.
 - DIS versions as from v2.0.4 require at least Microsoft Visual Studio 2012.
 - DIS versions as from v2.20.1 require at least Microsoft Visual Studio 2015.
 - DIS versions as from v2.35.1 require at least Microsoft Visual Studio 2017.
+- DIS versions as from v2.41 require at least Microsoft Visual Studio 2019.
```

Comment:
As we double checked recently and with Empower some confusion was raised, it could be interesting to specifically mention that Community edition, Professional & Enterprise version of Visual Studio can be used. The important one is Community as that is the free version of Visual Studio.